app/src/main/java/com/afollestad/mnmlscreenrecord/ui/settings/dialogs/TImePickerDialog.kt

```diff
@@ -64,7 +64,7 @@ class TimePickerDialog : DialogFragment() {
   override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
     val context = activity ?: blowUp()
     val args = arguments ?: blowUp()
-    val id = arguments?.getString(KEY_ID) ?: blowUp()
+    val id = args.getString(KEY_ID) ?: blowUp()
```

Comment:
` ?: blowUp()` can be removed from the end of this line 😉

Reply:
It actually can't, since getString() returns a nullable.